src/const/queries.js

```diff
@@ -2,3 +2,27 @@ export const APPLICATION = 'BLOCKCHAIN_APPLICATION';
 export const BLOCKS = 'GET_BLOCKS';
 export const BLOCKS_ASSETS = 'GET_BLOCKS_ASSETS';
 export const NETWORK_STATUS = 'GET_NETWORK_STATUS';
+export const LEGACY = 'GET_LEGACY';
+export const TOKENS_BALANCE = 'GET_TOKENS_BALANCE';
+export const TOKENS_SUPPORTED = 'GET_TOKENS_SUPPORTED';
+export const TOKENS_TOP_LSK_BALANCE = 'GET_TOKENS_TOP_LSK_BALANCE';
+export const PEERS = 'GET_PEERS';
+export const NEWSFEED = 'GET_NEWSFEED';
+export const DELEGATES = 'GET_DELEGATES';
+export const VALIDATOR = 'GET_FORGER_VALIDATOR';
+export const VOTES_SENT = 'GET_VOTES_SENT';
+export const VOTES_RECEIVED = 'GET_VOTES_RECEIVED';
+export const UNLOCKS = 'GET_UNLOCKS';
+export const MARKET_PRICES = 'GET_MARKET_PRICES';
+export const AUTH = 'GET_USER_INFO';
+export const GENERATOR = 'GET_FORGER_GENERATOR';
+export const TRANSACTIONS = 'GET_TRANSACTIONS';
+export const TRANSACTION_STATISTICS = 'GET_TRANSACTION_STATISTICS';
+export const NETWORK_STATISTICS = 'GET_NETWORK_STATISTICS';
+export const EVENTS = 'GET_TRANSACTION_EVENTS';
+export const BLOCKCHAIN_APPS = 'GET_ALL_BLOCKCHAIN_APPS';
+export const CCM = 'GET_CCM';
+export const COMMAND_PARAMETERS_SCHEMAS = 'GET_COMMAND_PARAMETERS_SCHEMAS';
+export const FEES = 'GET_FEES';
+export const BLOCKCHAIN_APPS_STATICS = 'GET_BLOCKCHAIN_APPS_STATICS';
+export const BLOCKCHAIN_APPS_META = 'GET_BLOCKCHAIN_APPS_META';
```

Comment:
The new names you have chosen might barrel into confusion. LEGACY is not much smaller or more readable than GET_LEGACY, but now developers need to follow the code pipeline to understand which method was used. 
Also, the new names are in some cases misleading. For example, GET_TRANSACTION_EVENTS is self explanatory in three aspects: The domain, the action, and the entity. but EVENTS is not.

Reply:
the Keys is the api name and the value has the more details
for example the api path is event but its actually is transaction we can take care the miss leading in own pr if has any